src/main/java/org/opentripplanner/gtfs/mapping/StopMapper.java

```diff
@@ -39,6 +45,27 @@ private Stop doMap(org.onebusaway.gtfs.model.Stop gtfsStop) {
         gtfsStop.getId().getAgencyId()
     );
 
+    if (translationHelper != null) {
+      return new Stop(base.getId(),
```

Comment:
I think we could just calculate the name and url in this block, and use the same constructor call for both cases? The same applies for all mappers